auto_round/auto_quantizer.py

```diff
@@ -169,16 +177,31 @@ def merge_quantization_configs(
         else:
             warning_msg = ""
 
+        loading_attr_dict = quantization_config_from_args.get_loading_attributes() \
+                if quantization_config_from_args is not None else None
         if isinstance(quantization_config, dict):
             if "auto-round" in quantization_config["quant_method"]:
                 quantization_config = AutoRoundConfig.from_dict(quantization_config)
             else:
-                quantization_config = AutoQuantizationConfig.from_dict(quantization_config)  # pylint: disable=E1101
+                if  ("use_auto_round_format" in quantization_config and quantization_config['use_auto_round_format']) \
+                        or ("use_auto_round_format" in loading_attr_dict and loading_attr_dict["use_auto_round_format"]):
+                    logger.info(f"Loading quantized model in auto_round format.")
+                    tmp_backend = quantization_config["quant_method"]
+                    target_backend = quantization_config["backend"] if "backend" in quantization_config else "auto"
+                    if loading_attr_dict is not None and "backend" in loading_attr_dict:
+                        target_backend = loading_attr_dict["backend"]
+                        loading_attr_dict.pop("backend")
```

Comment:
also could you help rename the target_backend to backend

Reply:
will be updated in next PR